models/library.json

```diff
@@ -0,0 +1,254 @@
+[
```

Comment:
What is this file? Maybe you could add a Readme in this directory

Reply:
Good idea! Doing...

Reply:
...done.